modules/account-lib/test/unit/coin/hbar/utils.ts

```diff
@@ -32,23 +21,17 @@ describe('HBAR util library', function() {
     });
   });
 
-  describe('transaction hash', function () {
-    it('should validate tx hashes', function () {
-      const validHashes = [
-        '0.0.14621@1595374723.356981689',
-      ];
+  describe('transaction hash', function() {
+    it('should validate tx hashes', function() {
+      const validHashes = ['0.0.14621@1595374723.356981689'];
 
       for (const hash of validHashes) {
         Utils.isValidTransactionHash(hash).should.be.true();
       }
     });
 
-    it('should fail to validate invalid tx hashes', function () {
-      const invalidHashes = [
-        '0.0.14621',
-        'invalid',
-        '',
-      ];
+    it('should fail to validate invalid tx hashes', function() {
+      const invalidHashes = ['0.0.14621', 'invalid', ''];
```

Comment:
same here re: hashes

Reply:
Oh but this is from another PR I think that it's shown here because of the change with the linter from " to '

Reply:
We can give it a look, though.
But maybe we should create a new PR for that.